scripts/release/tests/build_info_test.py

```diff
@@ -330,13 +330,15 @@ def test_load_build_info_release():
                 repositories=["quay.io/mongodb/mongodb-kubernetes-readinessprobe"],
                 platforms=["linux/arm64", "linux/amd64"],
                 dockerfile_path="docker/mongodb-kubernetes-readinessprobe/Dockerfile",
+                skip_if_exists=True,
                 olm_tag=True,
                 sign=True,
             ),
             "upgrade-hook": ImageInfo(
                 repositories=["quay.io/mongodb/mongodb-kubernetes-operator-version-upgrade-post-start-hook"],
                 platforms=["linux/arm64", "linux/amd64"],
                 dockerfile_path="docker/mongodb-kubernetes-upgrade-hook/Dockerfile",
+                skip_if_exists=True,
```

Comment:
q: why skip for readiness and upgrade hook and not for others?

Reply:
wouldn't skip prevent from rebuilding the latest tag?

Reply:
`skip_if_exists` is only used for `release` scenario, pushing to quay. The reason I only added this to readiness probe and version upgrade hook is because we don't bump their respective versions for each operator release, thus we don't want to override existing images in quay. For other operator related images that we always release it sometimes makes sense to rebuild and push new images i.e. when we need to retag the release because of some issues or rebuild agent images.

I think this question is interesting one and I would like to ask team about it. One downside is that the job succeeds, but nothing is pushed, which is not something we suspect is happening.

Reply:
i think the default should be always skip_if_exist for all images. Once we want to overwrite we should set explicitly set using to overwrite. This is safer IMO